config/settings.yml

```diff
@@ -1023,6 +1023,7 @@
   :interval: 60
   :memcache_server: 127.0.0.1:11211
   :memcache_server_opts: "-l 127.0.0.1"
```

Comment:
Just a thought... If we expose this, we should have a default  here  `:memcache_server_opts: "-l 127.0.0.1 -I 1m"` that matches the default below.  It would be nice to share the same value for each of these options so you don't have to update it in both places.  If memcached's -I option accepts bytes, it would be best to use the same unit.

cc @bdunne